notifications_utils/formatters.py

```diff
@@ -619,8 +623,45 @@ def add_language_divs(_content: str) -> str:
 def remove_language_divs(_content: str) -> str:
     """Remove the tags from content. This fn is for use in the email
     preheader, since this is plain text not html"""
-    content = re.compile(FR_OPEN).sub("", _content)
-    content = re.compile(FR_CLOSE).sub("", content)
-    content = re.compile(EN_OPEN).sub("", content)
-    content = re.compile(EN_CLOSE).sub("", content)
+    return remove_tags(_content, FR_OPEN, FR_CLOSE, EN_OPEN, EN_CLOSE)
+
+
+def add_ga_seal(_content: str) -> str:
+    """
+    Custom parser to add Global Affairs seal logo.
+
+    This is a custom temporary change not meant to exist for more than a few
+    weeks. This should either be removed or upgraded into a full-fledged
+    feature.
+
+    TODO: Review, remove/upgrade this functionality.
```

Comment:
Is there an associated bug to track this? (Or some other way to make sure it doesn't get lost?)

Reply:
I will create a clean up task that we can revisit in a future sprint.